source/classes.tex

```diff
@@ -6785,10 +6785,76 @@
 \rSec2[class.spaceship]{Three-way comparison}
 \indextext{operator!three-way comparison!defaulted}%
 
+\pnum
+The synthesized three-way comparison
+for comparison category type \tcode{R}\iref{cmp.categories}
+of glvalues \tcode{a} and \tcode{b} of the same type
+is defined as follows:
+
+\begin{itemize}
+\item
+If overload resolution for \tcode{a <=> b}
+finds a usable function \iref{over.match},
```

Comment:
Nitpik: Please remove preceding space before `\iref` for consistency.

Reply:
Done.